apierrors/errors.go

```diff
@@ -16,6 +16,14 @@ var (
 	ErrUnableToParseJSON   = errors.New("failed to parse json body")
 )
 
+var (
+	ErrDescription              = "Unable to process request due to a malformed or invalid request body or query parameter"
+	ErrInternalErrorDescription = "An internal error occurred"
+	ErrResourceNotFound         = "The requested resource does not exist"
+	ErrUnmarshallJSONObject     = "Failed to marshal bundle resource into bytes"
```

Comment:
```suggestion
	ErrUnmarshalJSONObject     = "Failed to marshal bundle resource into bytes"
```
It looks like we decided to use the spelling `unmarshal` throughout the repo so this should be consistent everywhere